packages/azure/components/main.json

```diff
@@ -0,0 +1,373 @@
+      }
+    },
+    "proxyPublicKey": {
+      "type": "string",
+      "metadata": {
+        "description": "Proxy enode public key"
+      }
+    },
+    "proxyPrivateKey": {
+      "type": "secureString",
+      "metadata": {
+        "description": "Proxy etherbase account private key"
+      }
+    },
+    "validatorPrivateKey": {
+      "type": "secureString",
+      "metadata": {
+        "description": "Validator etherbase account private key"
+      }
+    },
+    "validatorGethAccountSecret": {
+      "type": "secureString",
+      "metadata": {
+        "description": "Validator etherbase account password"
+      }
+    },
+    "attesterAccountAddress": {
+      "type": "string",
+      "metadata": {
+        "description": "Attester account address"
+      }
+    },
+    "attesterPrivateKey": {
+      "type": "secureString",
+      "metadata": {
+        "description": "Attester account private key"
+      }
+    },
+    "attesterGethAccountSecret": {
+      "type": "secureString",
+      "metadata": {
+        "description": "Attester account password"
+      }
+    },
+    "attesterTwilioAccountSID": {
+      "type": "string",
+      "metadata": {
+        "description": "Attester Twilio account SID"
+      }
+    },
+    "attesterTwilioAuthToken": {
+      "type": "secureString",
+      "metadata": {
+        "description": "Attester Twilio authentication token"
+      }
+    },
+    "attesterTwilioMessageServiceSID": {
+      "type": "string",
+      "metadata": {
+        "description": "Attester Twilio message service SID"
+      }
+    },
+    "bootnodes": {
+      "type": "string",
+      "defaultValue": "enode://5aaf10664b12431c250597e980aacd7d5373cae00f128be5b00364344bb96bce7555b50973664bddebd1cb7a6d3fb927bec81527f80e22a26fa373c375fcdefc@34.82.45.71:30301",
+      "metadata": {
+        "description": "geth --bootnodes argument"
+      }
+    },
+    "gethVerbosity": {
+      "type": "string",
+      "defaultValue": "3",
+      "metadata": {
+        "description": "geth --verbosity argument"
+      }
+    },
+    "celoEnv": {
+      "type": "string",
+      "defaultValue": "baklava",
+      "metadata": {
+        "description": "Celo network environment name"
+      }
+    },
+    "networkId": {
+      "type": "string",
+      "defaultValue": "200110",
+      "metadata": {
+        "description": "Numeric Celo network ID"
+      }
+    },
+    "gethNodeDockerImageRepository": {
+      "type": "string",
+      "defaultValue": "us.gcr.io/celo-testnet/celo-node",
+      "metadata": {
+        "description": "Celo geth Docker image repository"
+      }
+    },
+    "gethNodeDockerImageTag": {
+      "type": "string",
+      "defaultValue": "[parameters('celoEnv')]",
+      "metadata": {
+        "description": "Celo geth Docker image tag"
+      }
+    },
+    "gethExporterDockerImageRepository": {
+      "type": "string",
+      "defaultValue": "gcr.io/celo-testnet-production/geth-exporter",
```

Comment:
We haven't done any work to incorporate it into our own infrastructure yet, but because this isn't for our purposes and we've released the geth 1.9 merge, you're fine to just remove the geth exporter entirely from this PR